content/blog/nuttx-adding-porting-an-app/index.md

```diff
@@ -0,0 +1,471 @@
+---
+title: "Creating or Porting an Application to NuttX - Part 1"
+date: 2024-08-06T08:00:00-03:00
+tags: ["NuttX", "Apache", "ESP32", "POSIX", "Linux", "Tutorial"]
+showAuthor: false
+authors:
+    - "tiago-medicci"
+---
```

Comment:
If this is part 1, I suggest you create it as a series. Please add the following:

```
series: ["Porting-to-NuttX"]
series_order: 1
```

Reply:
Of course, I updated and rebased the article!